java-snapshot-testing-core/src/test/java/au/com/origin/snapshots/onLoadSnapshotFileTest.java

```diff
@@ -0,0 +1,67 @@
+package au.com.origin.snapshots;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import au.com.origin.snapshots.config.BaseSnapshotConfig;
+import au.com.origin.snapshots.serializers.ToStringSnapshotSerializer;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+public class onLoadSnapshotFileTest {
```

Comment:
Can you rename to `OnLoadSnapshotFileTest`, I'm not sure why `onSaveSnapshotFileTest.java` is not following conventions.  It needs to be cleaned up.

Reply:
Done